src/_pytest/logging.py

```diff
@@ -737,6 +753,10 @@ def emit(self, record):
                 self._section_name_shown = True
             super().emit(record)
 
+    def handleError(self, record: logging.LogRecord) -> None:
+        # Handled by LogCaptureHandler.
+        pass
```

Comment:
I suggest to change those `pass` to a docstring to "trick" coverage 😁 

```python
    def handleError(self, record: logging.LogRecord) -> None:
        """Handled by LogCaptureHandler."""
```

Reply:
Hehe I didn't know it would trick it.

Actually it really isn't covered so I prefer not to. I don't think it worth covering really, but if someone ever wants 100% coverage, they'd need to :)